apps/jobs/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.conf.urls import url
-# from .views import ()
 
+from . import views
 
 urlpatterns = [
+    url(r'challenge/(?P<challange_id>[0-9]+)/'
+        r'challenge_phase/(?P<challange_phase_id>[0-9]+)/',
```

Comment:
Yeah, we will add that functionality soon. This PR mainly focuses on creating the Submission  only since this is a major feature.